src/features/currentTodo.ts

```diff
@@ -28,8 +28,10 @@ const currentTodoReducer = (
   action: Action,
 ): State => {
   switch (action.type) {
-    // Implement all actions here
-
+    case 'currentTodo/SET':
```

Comment:
Better create Enum for action types too

Reply:
can we put objects into enum?

Reply:
> can we put objects into enum?

No, and you don't need to store objects in enum, it doesn't make sense